client/components/PriceSelector/PriceSelector.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './PriceSelector.css';
+
+export default class PriceSelector extends React.Component {
+  static choices = Array(4)
```

Comment:
this is not a selector why is it named selector...? also this priceselector is so specific basically its not reusable for anything else can you refactor this a bit please

Reply:
again, this should have tests

Reply:
Alright. didn't consider on creating a more generic component given the requirements of the tasks, but will extract a more generic component

Reply:
I currently have tests for price selector. What specific tests do you want me to add?

Reply:
this making array logic